spec/requests/api/v1/finders/customer_spec.rb

```diff
@@ -0,0 +1,149 @@
+require 'rails_helper'
+
+describe 'Customers API' do
+
+  let(:date) {"2012-03-27T14:54:02.000Z"}
+
+  context 'find by' do
+
+    let(:customer_1) { create(:customer) }
+    let(:customer_2) { create(:customer) }
+    let(:customer_3) { create(:customer) }
+
+    it 'id' do
+      id = customer_3.id
+
+      get '/api/v1/customers/find', params: {id: id}
+      customer = JSON.parse(response.body)
+
+      expect(response).to be_success
+      expect(customer['first_name']).to eq customer_3.first_name
+    end
+
+    it 'first_name' do
+      first_name = customer_3.first_name
+
+      get '/api/v1/customers/find', params: {first_name: first_name}
+      customer = JSON.parse(response.body)
+
+      expect(response).to be_success
+      expect(customer['id']).to eq customer_3.id
+    end
+
+    it 'last_name' do
+      last_name = customer_3.last_name
+
+      get '/api/v1/customers/find', params: {last_name: last_name}
+      customer = JSON.parse(response.body)
+
+      expect(response).to be_success
+      expect(customer['id']).to eq customer_3.id
+    end
+
+    it 'updated at' do
+      customer_3.update(updated_at: date)
+
+      get '/api/v1/customers/find', params: {updated_at: date}
+      customer = JSON.parse(response.body)
+
+      expect(response).to be_success
+      expect(customer['id']).to eq customer_3.id
+    end
+
+    it 'created at' do
+      customer_3.update(created_at: date)
+
+      get '/api/v1/customers/find', params: {created_at: date}
+      customer = JSON.parse(response.body)
+
+      expect(response).to be_success
+      expect(customer['id']).to eq customer_3.id
+    end
+
+    it 'multiple attributes' do
+      first_name = customer_3.first_name
+      last_name = customer_3.last_name
+      customer_3.update(created_at: date)
+      customer_3.update(updated_at: date)
+      id = customer_3.id
+
+      get '/api/v1/customers/find',
+      params: {first_name: first_name, last_name: last_name, id: id,created_at: date, updated_at: date}
```

Comment:
Thanks for breaking the `get` and the `params` onto separate lines.  I suppose if we wanted to, we could break out each of the key:value pair if we wanted to.